src/Protocol/DocumentUri.cs

```diff
@@ -14,17 +15,23 @@ namespace OmniSharp.Extensions.LanguageServer.Protocol
     /// <remarks>This exists because of some non-standard serialization in vscode around uris and .NET's behavior when deserializing those uris</remarks>
     public class DocumentUri : IEquatable<DocumentUri>
     {
+        public static Regex WindowsPath =
+            new Regex("^\\w(?:\\:|%3a)[\\\\|\\/]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+        public string _delimiter = SchemeDelimiter;
+
         /// <summary>
         /// Create a new document uri
         /// </summary>
-        /// <param name="url"></param>
+        /// <param name="url"></param> add .
         public DocumentUri(string url)
```

Comment:
Remove

Reply:
NO THE COMMENT STAGES THE CODE AUTOMATICALLY.... wait... nevermind! 😄

Reply:
Good for you it wasn't your password and just `git add .` that slipped :smile: